samples/.eslintrc.yml

```diff
@@ -1,4 +1,6 @@
 ---
 rules:
+  mocha: true
   no-console: off
   node/no-missing-require: off
+  node/no-unpublished-require: off
```

Comment:
I'm surprised you need this

Reply:
Remove this and run "npm i && npm run lint -- --fix", got below:

npm WARN acorn-jsx@5.0.1 requires a peer of acorn@^6.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN eslint-plugin-prettier@3.0.0 requires a peer of prettier@>= 1.13.0 but none is installed. You must install peer dependencies yourself.
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.2.4 (node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.4: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})

audited 7606 packages in 3.856s
found 0 vulnerabilities


> @google-cloud/asset-samples@0.0.1 lint ~/github/nodejs-asset/samples
> eslint '**/*.js' "--fix"


~/github/nodejs-asset/samples/system-test/quickstart.test.js
  20:23  error  "@google-cloud/nodejs-repo-tools" is not published  node/no-unpublished-require

✖ 1 problem (1 error, 0 warnings)

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/asset-samples@0.0.1 lint: `eslint '**/*.js' "--fix"`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @google-cloud/asset-samples@0.0.1 lint script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     ~/.npm/_logs/2018-11-27T17_53_16_244Z-debug.log

Reply:
Never mind, I realized that I was running lint against test file. Running it against dev file doesn't yield these errors.